google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -61,12 +59,29 @@ StorageIntegrationTest::~StorageIntegrationTest() {
   }
 }
 
+google::cloud::Options StorageIntegrationTest::MakeTestOptions(
+    google::cloud::Options opts) {
```

Comment:
nit: I would drop `google::cloud::` here and below

Reply:
Done.